src/pages/Settings.tsx

```diff
@@ -1,23 +1,77 @@
 import React from 'react';
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/react';
-import ExploreContainer from '../components/ExploreContainer';
+import {
+    IonHeader,
+    IonPage,
+    IonTitle,
+    IonToolbar,
+    IonRow,
+    IonButton,
+    IonIcon,
+    IonContent,
+    IonList,
+} from '@ionic/react';
+import { close } from 'ionicons/icons';
+
+/**
+ * Imports for all custom icons
+ */
+
+import './Settings.scss';
+import '../theme/Modal.scss';
+import '../theme/variables.scss';
+import { useDispatch } from 'react-redux';
+import { setPage } from '../features/Routing/RouterStore';
+import { Pages } from '../Models/Enums';
+import DarkMode from './SettingsPages/DarkMode';
 
 const Settings: React.FC = () => {
+    const dispatch = useDispatch();
+
     return (
         <IonPage>
             <IonHeader>
-                <IonToolbar>
-                    <IonTitle>Settings</IonTitle>
+                {/**
+                 * Blue toolbar with the title of the page: settings
+                 */}
+                <IonToolbar color="relectric-settings" className="SettingsToolBar">
+                    <IonRow>
+                        <IonButton
+                            fill="clear"
+                            color="white"
+                            shape="round"
+                            onClick={() => dispatch(setPage(Pages.Home))}
+                        >
+                            <IonIcon src={close} className="XButton" />
+                        </IonButton>
+                        <IonTitle>Settings</IonTitle>
+                    </IonRow>
                 </IonToolbar>
             </IonHeader>
-            <IonContent fullscreen>
-                <IonHeader collapse="condense">
-                    <IonToolbar>
-                        <IonTitle size="large">Settings</IonTitle>
-                    </IonToolbar>
-                </IonHeader>
-                <ExploreContainer />
+
+            <IonContent fullscreen={false} className="ModalContent">
+                <IonList lines="inset">
+                    <DarkMode />
+                </IonList>
             </IonContent>
+
+            {/* <IonContent className="ModalContent" fullscreen={false}>
```

Comment:
@brooke-k did you want to keep this block in for reference?

Reply:
This was just me playing around with methods for getting the settings page set up in a possibly better formatted matter. This block, however, is completely non-functioning and I honestly just forgot to get rid of it. It's been removed in the latest commit.